boms/cloud-oss-bom/RELEASING.md

```diff
@@ -62,11 +62,12 @@ new release is available on Maven Central.
 * Manually edit and update any pom.xml files in https://github.com/GoogleCloudPlatform/java-docs-samples that dependabot missed
     * Go to go/java-live
     * Sort by title
-    * Scroll down until you find the section with titles "chore(deps): update dependency com.google.cloud:libraries-bom to v<version>"
-   * Approve and merge these PRs.
+    * Scroll down until you find the section with titles "chore(deps): update dependency com.google.cloud:libraries-bom to v&lt;version>"
+    * Approve and merge these PRs.
+      For PRs waiting for Kokoro jobs, add the "kokoro:force-run" label.
 * In google3 run:
     * `$ g4d -f bom`
-    *  `/google/src/head/depot/google3/devtools/scripts/replace_string "&lt;version>oldVersion&lt;/version>" "&lt;version>newVersion&lt;/version>"`
+    *  `/google/src/head/depot/google3/devtools/scripts/replace_string "&lt;version&gt;oldVersion&lt;/version&gt;" "&lt;version&gt;newVersion&lt;/version&gt;"`
```

Comment:
stet. These should be '>' per the Google HTML style guide

Reply:
Until we can convince the owner to follow the Google HTML style, shall we follow the owner's guide to use `&gt;`?

Reply:
There are many documents and many owners, who do not always agree. One purpose of the style guide is to resolve these disagreements once and for all rather than switching back and forth in every other cl to satisfy the individual preference of the reviewer.

Reply:
I see the latest CL by you fixed this. Reverted this line.